RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernelsImpl.h

```diff
@@ -78,9 +78,9 @@ __global__ void kernel_checkOverflows(HitContainer const *foundNtuplets,
   }
 
   for (int idx = first, nt = foundNtuplets->nOnes(); idx < nt; idx += gridDim.x * blockDim.x) {
-    if (foundNtuplets->size(idx) > 5)
+    if (foundNtuplets->size(idx) > 7)  // current real limit
       printf("ERROR %d, %d\n", idx, foundNtuplets->size(idx));
-    assert(foundNtuplets->size(idx) < 6);
+    assert(foundNtuplets->size(idx) <= caConstants::maxHitsOnTrack);
```

Comment:
so
  - `size == 7` is the largest that should happen in the current algorithms
  - `size == caConstants::maxHitsOnTrack` (currently `10`) is the largest that can fit in the data structures
  
?

Reply:
yes